test_files/functions/two_jsdoc_blocks.js

```diff
@@ -0,0 +1,22 @@
+goog.module('test_files.functions.two_jsdoc_blocks');var module = module || {id: 'test_files/functions/two_jsdoc_blocks.js'};/**
+ * @fileoverview This text here matches the     text below in length.
```

Comment:
This comment looks wrong.  Both of the functions seems to have Jsdoc comments that are shorter than this.

Reply:
The second function's document is exactly 77 chars, as is this one.